### DISCUSSION TITLE
The ML framework you used is not what you claimed

### DISCUSSION THREAD

**Discussion by Chaodong** [PaddlePaddle/PaddleSpatial#135] [BaiduKDDCup2022] [answered]
my model is based on paddlepaddle, the same as I claimed, but got this error
I think it is the problem of evaluation code:
![image](https://user-images.githubusercontent.com/1487440/171542921-f0f6fa62-d38a-4e81-9e51-0802c022f43f.png)

**Comment by ramber1836** [comment]
Please provide your team name and submission time.

**Reply by Chaodong** [reply]
Variational_singula at 2022-06-02 10:22

**Comment by ramber1836** [comment, marked as answer]
> my model is based on paddlepaddle, the same as I claimed, but got this error I think it is the problem of evaluation code: ![image](https://user-images.githubusercontent.com/1487440/171542921-f0f6fa62-d38a-4e81-9e51-0802c022f43f.png)

The online test has been updated. You can try it again.
